SQLTask/Task7.sql

```diff
@@ -0,0 +1,75 @@
+USE DB_Bank
+
+GO
+CREATE PROCEDURE TransferMoney
+	@accountName VARCHAR(30),
+	@cardNumber VARCHAR(19),
+	@moneyCount MONEY
+AS
+	BEGIN
+		DECLARE @totalCardBalance MONEY, 
+			@accountBalance MONEY
+
+		SET @totalCardBalance = 

```

Comment:
Change SP to avoid repeatable read issue. Read more here 
https://professorweb.ru/my/sql-server/2012/level3/3_16.php

Reply:
Resolve